src/main/java/com/ft/methodearticleinternalcomponentsmapper/transformation/MethodeLinksBodyProcessor.java

```diff
@@ -177,59 +172,49 @@ private boolean isRemovable(final Node aTag) {
         return true;
     }
 
-    private List<String> getUuidsPresentInContentStore(Set<String> uuidsToCheck, String transactionId) {
-        if (uuidsToCheck.isEmpty()) {
-            return Collections.emptyList();
+    private Content[] getContentFromDocumentStore(Map<Node, String> tags, String transactionId) {
```

Comment:
It would be nicer to return a `List<Content>` instead of an array, and along the flow to use a List or a Set.